avatar/avatar.css

```diff
@@ -0,0 +1,189 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&family=Merriweather:wght@300&family=Montserrat&family=Nanum+Gothic&family=Noto+Serif&family=Quicksand&family=Raleway:wght@200&family=Roboto+Slab:wght@300&family=Rubik:wght@300&family=Source+Sans+Pro&family=Ubuntu:wght@300&family=Work+Sans:wght@200&display=swap");
```

Comment:
A suggestion here. You can center the nav items. as per my view it will look even better